docs/guides/accelerate_question_answering.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 2
+slug: /accelerate_question_answering
+---
+
+# Accelerate document indexing and retrieval
+
+A checklist to speed up document parsing and question answering.
+
+---
+
+Please note that several of your settings may *significantly* increase the time required for document parsing and retrieval. If you often find that document parsing and question answering are time-consuming, here is a checklist to consider:
+
+1. Use GPU to reduce embedding time.
```

Comment:
Two chapter:
1. Accelerate document indexing
2. Accelerate question answer, but not retrieval